tslearn/barycenters/dba.py

```diff
@@ -176,7 +178,7 @@ def dtw_barycenter_averaging_petitjean(X, barycenter_size=None,
     return barycenter
 
 
-def _mm_assignment(X, barycenter, weights, metric_params=None):
+def _mm_assignment(X, barycenter, weights, metric_params=None, n_treads=15):
```

Comment:
We use `n_jobs` everywhere as a parameter name, and `None` as its default, we should stick to the same convention here, I think.